File: samples/search/php/update-track.php
<?php

$curl = curl_init();

$apiKey = "";    // Replace with your API Key. Required
$track_id = "";  // Replace with the track ID. Required
$customer_track_id = "";
$primary_customer_track_id = "";
$version = "";
$title = "";
$album = "";
$artist = "";
$year = "";
$region = "";
$language = "";
$explicit = "";
$genre = "";
$moods = "";
$style = "";
$instruments = "";
$description = "";
$keywords = "";

$requestBody = array(
  'id' => $track_id,
  'customer_track_id' => $customer_track_id,
  'primary_customer_track_id' => $primary_customer_track_id,
  'version' => $version,
  'title' => $title,
  'album' => $album,
  'artist' => $artist,
  'year' => $year,
  'region' => $region,
  'language' => $language,
  'explicit' => $explicit,
  'genre' => $genre,
  'moods' => $moods,
  'style' => $style,
  'instruments' => $instruments,
  'description' => $description,
  'keywords' => $keywords
);

$authHeader = $apiKey.":"."";
$encodedAuthHeader = base64_encode($authHeader);

curl_setopt_array($curl, array(
  CURLOPT_URL => "https://api-us.musiio.com/v1/catalog/track",
  CURLOPT_RETURNTRANSFER => true,
  CURLOPT_ENCODING => "",
  CURLOPT_MAXREDIRS => 10,
  CURLOPT_TIMEOUT => 0,
  CURLOPT_FOLLOWLOCATION => true,
  CURLOPT_HTTP_VERSION => CURL_HTTP_VERSION_1_1,
  CURLOPT_CUSTOMREQUEST => "PUT",
  CURLOPT_POSTFIELDS => $requestBody,
  CURLOPT_HTTPHEADER => array(
    "Authorization: Basic ".$encodedAuthHeader
  ),
));

$response = curl_exec($curl);

curl_close($curl);
echo $response;
?>

File: samples/search/java/UpdateTrack.java
package com.example.search;     // Replace it with your own project group ID

import org.springframework.http.HttpEntity;
import org.springframework.http.HttpHeaders;
import org.springframework.http.HttpMethod;
import org.springframework.http.MediaType;
import org.springframework.http.ResponseEntity;
import org.springframework.util.LinkedMultiValueMap;
import org.springframework.util.MultiValueMap;
import org.springframework.web.client.RestTemplate;

public class UpdateTrack {
    private static final String TRACK_ID = "";    // Track ID here
    private static final String API_KEY = "";       // Your API key
    private static final String URL = "https://api-us.musiio.com/v1/catalog/track";
    public static void main(String[] args) {
        HttpHeaders headers = HeadersUtils.createHeaders(API_KEY, "");
        headers.setContentType(MediaType.MULTIPART_FORM_DATA);
        MultiValueMap<String, Object> requestBody = new LinkedMultiValueMap<>();
        requestBody.add("id", TRACK_ID);
        requestBody.add("primary_customer_track_id", "");
        requestBody.add("version", "");
        requestBody.add("title", "");
        requestBody.add("album", "");
        requestBody.add("artist", "");
        requestBody.add("year", "");
        requestBody.add("region", "");
        requestBody.add("language", "");
        requestBody.add("explicit", "");
        requestBody.add("genre", "");
        requestBody.add("moods", "");
        requestBody.add("style", "");
        requestBody.add("instruments", "");
        requestBody.add("description", "");
        requestBody.add("keywords", "");
        HttpEntity<MultiValueMap<String, Object>> requestEntity = new HttpEntity<>(requestBody, headers
        );
        RestTemplate restTemplate = new RestTemplate();
        ResponseEntity<String> response = restTemplate.exchange(URL, HttpMethod.PUT, requestEntity, String.class);
        System.out.println(response.getBody());
    }
}
